pkg/controller/migmigration/backup.go

```diff
@@ -271,12 +346,14 @@ func (t *Task) bounceResticPod() error {
 }
 
 // Annotate all resources with PV action data
-func (t *Task) annotateStorageResources() error {
+// Return the number of copy pods so that we know if we need to create stage
```

Comment:
What are copy pods and what determines how many of them we'll end up with?

P.S. I know stage pods are also called dummy pods, it would be good to unify the language across this PR if it hasn't happened already.

Reply:
I'm seeing that a "restic pod" == "copy pod" now, but still don't know for sure what either of these are. I can assume that restic pods are the number of restic pods in the daemonset but want to verify that with you.